modules/apps/segments/segments-lang/src/main/resources/content/Language.properties

```diff
@@ -62,6 +62,7 @@ no-users-have-been-assigned-to-this-segment=No users have been assigned to this
 not-contains=Not Contains
 not-equals=Not Equals
 preview-members=Preview Members
+segment-name=Segment Name
```

Comment:
the placeholder should be "Untitled Segment" to be more consistent with other parts of the portal? for example, Web Content? (Untitled Basic Web Content)

also, the lang key `unnamed-segment=Unnamed Segment` is not in use anymore, so delete, please :)